packages/platform_maps_flutter_google_android/lib/src/google_map_original.dart

```diff
@@ -0,0 +1,3 @@
+/// These files are literal copies of [google_maps_flutter](https://github.com/flutter/packages/tree/main/packages/google_maps_flutter/google_maps_flutter/lib/src)
```

Comment:
Does the license play well with this inclusion?

Reply:
Is source inclusion needed? Or can you have a pre-compile build step that pulls in this dependency's sources (with a pinned version)?

Reply:
I basically want `google_maps_flutter`, but without the dependencies on `google_maps_flutter_android` and `google_maps_flutter_ios` (especially iOS). Getting the `google_maps_flutter_platform_interface` is really easy, but we do need the little bit of code that's in `google_maps_flutter` as well. I could ask them to move it to the platform interface, but no idea if they are willing to.

I wonder, would it be ok to copy the copyright license file in the `src/google_maps_original` directory?

Reply:
> I wonder, would it be ok to copy the copyright license file in the src/google_maps_original directory?

This should be possible. Keep the copied files in a folder by themselves and add the original license to that folder. @koozz?